server_modules/address.js

```diff
@@ -0,0 +1,20 @@
+var os = require('os');
+var args = require('./args')
+
+//shamelessly copied from StackOverflow with only minor changes
```

Comment:
Although this is simple code, copying stuff without a clear license may conflict with the license of this project.